### ISSUE TITLE
strange loss records

### ISSUE THREAD

**Issue by fyb99** [RobinWu218/SimDINO#49]
[log.txt](https://github.com/user-attachments/files/23438946/log.txt) Hi, i tried your MCR loss on my own dataset. The attached are the training logs. i find the training loss is increaseing and then decreasing, since it is a negative value. Why the loss such a trend, do you have any ideas?

**Comment by RobinWu218** [comment]
Hi, this is expected behavior because the compression loss (MSE) tends to increase slightly before decreasing again during training, as you can see in our [example log](https://github.com/RobinWu218/SimDINO/blob/main/assets/SimDNIOv1_vitb16.txt).

**Comment by fyb99** [comment]
Hi, but i find your comp loss is increasing, while mine is decreasing

**Comment by RobinWu218** [comment]
I see. In your case, the comp loss starts at almost 1, suggesting that you start from a almost collapsed/degenerate solution. I suggest you take a look at your initialization scheme. In our experience, starting from a non-collapsed solution is better for optimization and the final performance.

**Comment by fyb99** [comment]
I am little confused, so you mean i should change the initialization strategy?

**Comment by RobinWu218** [comment]
In our experiments, you can see that the model did not start from a collapsed solution. Did you use the same setting as ours?
